### PR TITLE
Feature/output time

### PR DESCRIPTION
**Pull request by prescod**
# Changes

CumulusCI now reports how long it took for flows to run.

For example:

2021-01-05 15:05:39: Completed flow 'config_qa' successfully!
2021-01-05 15:05:39: config_qa: Ran in 22s

2021-01-05 15:04:57: Completed flow 'dependencies' successfully!
2021-01-05 15:04:57: dependencies: Ran in 1h:2m:3s

2021-01-05 15:03:19: Completed flow 'robot_docs' successfully!
2021-01-05 15:03:19: robot_docs: Ran in 7s

### CODE REVIEW

**Review by davisagli — APPROVED** [review]
It's divmod's time to shine!

File: cumulusci/core/tests/test_utils.py [hunk 1]
```diff
@@ -167,6 +167,21 @@ def test_cleanup_cache_dir_nothing_to_cleanup(self):
                     cleanup_org_cache_dirs(keychain, project_config)
                     assert not rmtree.mock_calls, rmtree.mock_calls
 
+    duration = (
+        (59, "59s"),
+        (70, "1m:10s"),
+        (119, "1m:59s"),
+        (65, "1m:5s"),
+        (4000, "1h:6m:40s"),
+        (4000, "1h:6m:40s"),
```
Comment (Br4nd0R) [inline cumulusci/core/tests/test_utils.py:176]: Is this a duplicate on purpose?

Comment (prescod) [inline cumulusci/core/tests/test_utils.py:176]:
```suggestion
```

Comment (prescod) [reply cumulusci/core/tests/test_utils.py]: No. Fixed now.

**Review by davisagli — APPROVED** [review]

**Pull request merged**